lib/api/unittest/CBoostedTreeInferenceModelBuilderTest.cc

```diff
@@ -256,7 +256,7 @@ void CBoostedTreeInferenceModelBuilderTest::testIntegrationClassification() {
 }
 
 void CBoostedTreeInferenceModelBuilderTest::testJsonSchema() {
-    std::size_t numberExamples = 1000;
+    std::size_t numberExamples = 200;
```

Comment:
Why is this change in this PR? Is it somehow related?

Reply:
No you a right, it's unrelated, I will move it to another PR.